ansible/templates/etc/php/7.2/fpm/pool.d/pool-template.conf.j2

```diff
@@ -0,0 +1,18 @@
+; {{ ansible_managed }}
+
+[{{ item.name }}]
+user = {{ item.user }}
+group = {{ item.user }}
+listen = /var/run/php/php-fpm-{{ item.name }}.sock
+listen.owner = www-data
+listen.group = www-data
+php_admin_value[disable_functions] = exec,passthru,shell_exec,system
+php_admin_flag[allow_url_fopen] = off
+pm = dynamic
+pm.max_children = 5
+pm.start_servers = 2
+pm.min_spare_servers = 1
+pm.max_spare_servers = 3
+
+; Would be nice, but not very straightforward to implement
```

Comment:
This means the process can only access this dir? What's blocking this?

Reply:
Yes, but it also breaks some PHP functions without taking special care. We might make this work, but this PR is large enough as it is, so I'm filing this under "future improvements".

As an alternative (which has other benefits), I'm looking at tightening the permissions of "sensitive directories" (e.g. `/var/www`) using FACL's. I'll put that on a new branch as well.